rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java

```diff
@@ -168,5 +168,8 @@ public BridgeDevNetConstants(List<BtcECKey> federationPublicKeys) {
         maxInputsPerPegoutTransaction = 10;
 
         numberOfBlocksBetweenPegouts = 360; // 3 hours of RSK blocks (considering 1 block every 30 seconds)
+
+        estimatedPegoutTxIndexBtcActivationHeight = 1; // TODO: TBD. This is an estimation of the btc block number once RSKIP379 is activated.
+        pegoutTxIndexGracePeriodInBtcBlocks = 500; // TODO: TBD. Period of grace in btc blocks to continue using the legacy mechanism to identify btc tx.
```

Comment:
Will These Values Be Different Or Same For Each Networks?

Reply:
They will be different for each network.